hugegraph-core/src/main/java/org/apache/hugegraph/backend/tx/GraphTransaction.java

```diff
@@ -1574,14 +1597,14 @@ private void checkNonnullProperty(HugeVertex vertex) {
         // Check whether passed all non-null property
         @SuppressWarnings("unchecked")
         Collection<Id> nonNullKeys = CollectionUtils.subtract(
-                                     vertexLabel.properties(),
-                                     vertexLabel.nullableKeys());
+                vertexLabel.properties(),
+                vertexLabel.nullableKeys());
         if (!keys.containsAll(nonNullKeys)) {
             @SuppressWarnings("unchecked")
             Collection<Id> missed = CollectionUtils.subtract(nonNullKeys, keys);
             HugeGraph graph = this.graph();
             E.checkArgument(false, "All non-null property keys %s of " +
-                            "vertex label '%s' must be set, missed keys %s",
+                                   "vertex label '%s' must be set, missed keys %s",
```

Comment:
prefer to wrap line before "All non-null..."

Reply:
fixed